applications/crossbar/src/modules/cb_allotments.erl

```diff
@@ -124,19 +124,22 @@ load_consumed(Context) ->
 
 -spec foldl_consumed(api_binary(), wh_json:object(), {cb_context:context(), {'cycle', wh_datetime()} | {'manual', api_seconds(), api_seconds()} , wh_json:objects()}) ->
     {cb_context:context(), {'cycle', wh_datetime()} | {'manual', api_seconds(), api_seconds()} , wh_json:objects()}.
-foldl_consumed(_Classification, _Value, {#cb_context{resp_status='error'}=ContextErr, Mode, Acc}) -> {ContextErr, Mode, Acc};
 foldl_consumed(Classification, Value, {Context, Mode, Acc}) ->
-    case create_viewoptions(Context, Classification, Value, Mode) of
-        {Cycle, ViewOptions} ->
-            [_, From] = props:get_value('startkey', ViewOptions),
-            [_, To] = props:get_value('endkey', ViewOptions),
-            ContextResult = crossbar_doc:load_view(?LIST_CONSUMED
-                                                   ,props:insert_value({'group_level', 1},ViewOptions)
-                                                   ,Context
-                                                  ),
-            Acc1 = normalize_result(Cycle, From, To, Acc, cb_context:doc(ContextResult)),
-            {ContextResult, Mode, Acc1};
-        ContextErr -> {ContextErr, Mode, Acc}
+    case cb_context:resp_status(Context) of
+        'error' -> {Context, Mode, Acc};
```

Comment:
Why not match `'success'` and do the clause, and use catchall for this clause? `resp_status` could be `'fatal'` too.